packages/teleterm/src/mainProcess/windowsManager.ts

```diff
@@ -41,6 +41,10 @@ export class WindowsManager {
       title: 'Teleport Connect Preview',
       icon: getAssetPath('icon.png'),
       webPreferences: {
+        devTools: this.settings.dev,
+        webgl: false,
+        enableWebSQL: false,
+        safeDialogs: true,
```

Comment:
What were the errors you were running into when `sandbox` was set to `true`?

From reading the docs alone it seems that sandbox should just work for us but obviously I'm missing something here.

Reply:
The `loggerService` would start to become undefined causing the app the break. Removing it would cascade into other problems. Didn't follow it to the end but that was the thread I started pulling before thinking it could just be a new issue

Reply:
It can be caused by my changes here: https://github.com/gravitational/webapps/pull/853#discussion_r887943357 :/ 
Maybe we should think about it one more time?

Reply:
Oh I see, this might be a problem with Winston being a fundamentally a node package which requires node APIs. Inside renderers, any interaction with Winston would have to be done through IPC/gRPC, I guess any Winston objects could live only in the main and the shared process.

That's definitely a bigger change, I'll create a task for it just to have a dedicated place in case we want to come back to this in the future.